openpype/hosts/houdini/plugins/publish/save_scene.py

```diff
@@ -19,7 +19,7 @@ def process(self, context):
             "Collected filename from current scene name."
         )
 
-        if host.has_unsaved_changes():
```

Comment:
This isn't necessarily needed to fix the issue - but makes it forward compatible (since the old method is deprecated) so this change has to be done some time anyway.